### PR TITLE
Define magnetic field parameters for Geant4e as in Geant4  

### PR DESCRIPTION
**Pull request by civanch**
#### PR description:
For Geant4e propagator use the same parameters as established for Geant4 tracking in field at SIM step in (#32063 and #31665)

#### PR validation:
private

### CODE REVIEW

**Comment by civanch** [comment]
please test

File: TrackPropagation/Geant4e/python/geantRefit_cff.py [hunk 1]
```diff
@@ -20,12 +20,12 @@
                  Type = cms.string('CMSIMField'),
                  StepperParam = cms.PSet(
                      MaximumEpsilonStep = cms.untracked.double(0.01), ## in mm
-                     DeltaOneStep = cms.double(0.001), ## in mm
+                     DeltaOneStep = cms.double(1e-4),## in mm
                      MaximumLoopCounts = cms.untracked.double(1000.0),
```
Comment (mmusich) [inline TrackPropagation/Geant4e/python/geantRefit_cff.py:24]: @civanch,
when trying to fix the Geant4e refitter configuration in 11.2.x for PR https://github.com/cms-sw/cmssw/pull/32260 
I had to add the following parameters:
```python
    DeltaChordTracker = cms.double(0.001), ## in mm
    DeltaOneStepTracker = cms.double(1e-4),## in mm
    DeltaIntersectionTracker = cms.double(1e-6),## in mm
    RmaxTracker = cms.double(8000),        ## in mm
    ZmaxTracker = cms.double(11000),       ## in mm
    EnergyThTracker = cms.double(0.2),     ## in GeV
```
shouldn't those be added here as well?

**Comment by civanch** [comment]
+1

**Comment by silviodonato** [comment]
+1
@civanch see https://github.com/cms-sw/cmssw/pull/32268#discussion_r530214299

**Pull request merged**